src/libraries/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -16,6 +16,9 @@ internal static partial class NetSecurityNative
             IntPtr bufferPtr,
             ulong length);
 
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint = "NetSecurityNative_EnsureGssInitialized")]
```

Comment:
You probably want to use 'GeneratedDllImport' like the others in the file.

Reply:
@JRahnama  for context, this is new in 7.0. So you will need to use the new form in this change, and the back port will use the form you have here.